client/src/views/header/HeaderPage.vue

```diff
@@ -94,6 +94,29 @@
                 :icon="search"
               />
             </ion-button>
+            <div
+              v-if="!isMobile() && securityWarningsCount > 0 && isSmallDisplay"
+              id="trigger-checklist-button"
+              class="topbar-right-button__item unread"
+              @click="openSecurityWarningsModal()"
+              ref="checklistSecurityButton"
+            >
+              <ion-icon :icon="checkmarkCircle" />
+              <div class="checklist-security-levels">
+                <span
+                  class="security-level"
+                  :class="{ 'security-level--done': securityWarningsCount < 3 }"
+                />
```

Comment:
For non-workspace owners, you can add this condition on the first `span`
```suggestion
                <span
                  v-if="securityWarnings.isWorkspaceOwner"
                  class="security-level"
                  :class="{ 'security-level--done': securityWarningsCount < 3 }"
                />
```

The style will need tweaking